src/main/java/org/geppetto/simulator/remote/services/NSGUtilities.java

```diff
@@ -0,0 +1,145 @@
+package org.geppetto.simulator.remote.services;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.zip.GZIPInputStream;
+
+import org.apache.commons.compress.archivers.tar.TarArchiveEntry;
+import org.apache.commons.compress.archivers.tar.TarArchiveInputStream;
+import org.geppetto.core.common.GeppettoExecutionException;
+import org.ngbw.directclient.CiCipresException;
+import org.ngbw.directclient.CiClient;
+import org.ngbw.directclient.CiJob;
+
+/**
+ * Utility class for services. 
+ * 
+ * @author jrmartin
+ *
+ */
+public class NSGUtilities {
+
+	public static CiJob sendJob(CiClient myClient, String jobName, Path filePath, boolean validateOnly) throws CiCipresException, IOException, GeppettoExecutionException, InterruptedException
+	{
+		CiJob jobStatus;
+		
+		Map<String, Collection<String>> vParams = new HashMap<String, Collection<String>>();
+		HashMap<String, String> inputParams = new HashMap<String, String>();
+		HashMap<String, String> metadata = new HashMap<String, String>();
+
+		inputParams.put("infile_", filePath.toString());
+
+		// AQP: What should we be sending as parameters?
+		// See https://www.phylo.org/restusers/docs/guide.html#UseOptionalMetadata for list of available
+		metadata.put("statusEmail", "true");
+		// metadata.put("clientJobName", jobName);
+		metadata.put("clientJobId", "1234546");
```

Comment:
@tarelli I have just changed. We are going to be using the experiment id for the moment... I have removed the comments. They were for myself however for the next release I would like to add a few more parameter regarding metadata and parallelization. Anyway, this version works nicely and is stable enough to be merged.